test/benchmark/fixtures.rb

```diff
@@ -27,6 +27,15 @@ class PostSerializer < ActiveModel::Serializer
   belongs_to :blog, serializer: BlogSerializer
   belongs_to :author, serializer: AuthorSerializer
 
+  link(:post_authors) { 'https://example.com/post_authors' }
+
+  meta do
+    {
+      rating: 5,
+      favorite_count: 10
+    }
+  end
```

Comment:
taking this to the extreme, we should make the attributes/serializer ratio even higher to elimiate noise. something with like 100+ attributes would be a good smoke test.